src/sentry/runner/commands/cleanup.py

```diff
@@ -168,11 +136,21 @@ def cleanup(days, project, concurrency, max_procs, silent, model, router, timed)
         click.echo('Error: Minimum concurrency is 1', err=True)
         raise click.Abort()
 
-    import math
-    import multiprocessing
-    import pickle
-    import subprocess
-    import sys
+    pool = []
+    # Make sure we fork off multiprocessing pool
+    # before we import or configure out app
+    from multiprocessing import Process, JoinableQueue
+
+    task_queue = JoinableQueue(1000)
+    for _ in xrange(concurrency):
+        p = Process(target=multiprocess_worker, args=(task_queue,))
+        p.daemon = True
+        p.start()
+        pool.append(p)
+
+    from sentry.runner import configure
+    configure()
```

Comment:
TIL. Could have used this for my sweet forking solution.

Reply:
:) Yeah, this just punts all of the stuff that matters post fork.